File: eureka/ios/nativeLib/nativeLib/Models/AMember.swift
//
//  AMember.swift
//  LifePlus
//
//  Created by work on 10/09/20.
//

import Foundation

public class AMember {
  public final var name: String
  public final var dataTypeEnum: DataTypeEnum
  public final var length: Int

  public init(pName: String, pDataType: DataTypeEnum, pLength: Int) {
        name = pName
        dataTypeEnum = pDataType
        length = pLength
    }
}

File: eureka/android/library/src/main/java/com/lifeplus/Pojo/AMember.java
package com.lifeplus.Pojo;

import com.lifeplus.Pojo.Enum.DataTypeEnum;

public class AMember {
    public final String name;
    public final DataTypeEnum dataType;
    public final int length;

    public AMember(String pName, DataTypeEnum pDataType, int pLength) {
        name = pName;
        dataType = pDataType;
        length = pLength;
    }
}
